stac_fastapi/pgstac/app.py

```diff
@@ -127,9 +157,7 @@ async def lifespan(app: FastAPI):
 api = StacApi(
     app=update_openapi(fastapp),
     settings=settings,
-    extensions=extensions + [collection_search_extension]
-    if collection_search_extension
-    else extensions,
+    extensions=application_extensions,
```

Comment:
why do you want to limit this list to just the `application_extensions`. will that affect the advertised conformance classes?

Reply:
the application_extensions has all the extensions (transaction, search, collection_search, which we add only if available): 
https://github.com/stac-utils/stac-fastapi-pgstac/blob/5ef23dcc629700a47bf902385084b8b1d1904a0e/stac_fastapi/pgstac/app.py#L96-L100

https://github.com/stac-utils/stac-fastapi-pgstac/blob/5ef23dcc629700a47bf902385084b8b1d1904a0e/stac_fastapi/pgstac/app.py#L110

https://github.com/stac-utils/stac-fastapi-pgstac/blob/5ef23dcc629700a47bf902385084b8b1d1904a0e/stac_fastapi/pgstac/app.py#L137